src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -79,10 +79,6 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Include="/NATVIS:&quot;$(MSBuildThisFileDirectory)CoreRTNatVis.natvis&quot;" />
     </ItemGroup>
 
-    <ItemGroup>
-      <!-- TODO <LinkerArg Include="/MACHINE:X64" /> -->
```

Comment:
The linker infers the machine type from the `.obj` file we generated, which goes first on the command line.